ludwig/models/modules/sequence_decoders.py

```diff
@@ -316,6 +283,7 @@ def call(
         except KeyError:
             encoder_output_state = None
 
+        # todo tf2 need to move this to sequence output feature class
```

Comment:
mmm i don-t think so, the attention mechanism is a generator specific thing, it should live in the generator class